src/pages/home/report/ReportActionItem.js

```diff
@@ -593,6 +599,9 @@ export default compose(
         preferredSkinTone: {
             key: ONYXKEYS.PREFERRED_EMOJI_SKIN_TONE,
         },
+        iouReport: {
+            key: ({report}) => `${ONYXKEYS.COLLECTION.REPORT}${report.iouReportID}`,
```

Comment:
There is one if the report has an associated iouReport. Otherwise, none of this would work:

<img width="583" alt="Screenshot 2023-08-08 at 11 31 36 AM" src="https://github.com/Expensify/App/assets/19366280/7c0955a5-654c-4426-b9a9-4ff1bb7115a3">